examples/react-native/src/features/Authenticator/Federated/aws-exports.js

```diff
@@ -0,0 +1,10 @@
+import awsExports from '@aws-amplify/ui-environments/auth/auth-with-federated/src/aws-exports';
+
+export default {
+  ...awsExports,
+  oauth: {
+    ...awsExports.oauth,
+    redirectSignIn: 'myapp://ui/components/authenticator/sign-in-federated/',
+    redirectSignOut: 'myapp://ui/components/authenticator/sign-in-federated/',
+  },
+};
```

Comment:
The paths following the scheme do not provide any benefit since there is no routing in the example app
```suggestion
export default awsExports;
```